app/views/abouts/show.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="font-karla flex flex-col items-center w-full">
+    <div class="bg-blue-marine flex flex-col items-center px-5 pt-6 pb-10 w-full">
+        <div class="max-w-screen-sm text-center font-normal">
+            <p class="text-base mb-6 text-white">Brought to you by</p>
+            <%= inline_svg_tag "telos_labs_logo.svg", class: "mb-10 w-full" %>
+            <p class="text-white text-lg mb-6">We designed and developed this open-source app for the Rails community, you can contribute to the repo <a href="https://github.com/TelosLabs/rails-world" target="_blank" class="text-blue-teal">here</a>.</p>
+            <p class="text-white text-lg mb-10">Telos creates disruptive, purpose-driven software for visionary clients to enhance productivity and enrich lives.</p>
+            <div class="flex flex-row w-full items-center gap-5">
+                <a href="https://hi.teloslabs.co/" target="_blank" class="grow bg-blue-teal rounded-[10px] p-4 flex flex-row items-center justify-center">
+                    <%= inline_svg_tag "icon_web.svg", class: "w-6 h-6" %>
+                </a>
+                <a href="https://www.linkedin.com/company/telos-labs" target="_blank" class="grow bg-blue-teal rounded-[10px] p-4 flex flex-row items-center justify-center">
+                    <%= inline_svg_tag "icon_linkedin.svg", class: "w-6 h-6" %>
+                </a>
+            </div>
+        </div>
+    </div>
+    <div class="bg-blue-marine-light py-10 px-5 flex flex-col items-center w-full">
+        <div class="max-w-screen-sm text-center font-normal w-full">
+            <h2 class="text-blue-teal text-[25px] mb-10">
+                Credits
+            </h2>
+            <div class="flex flex-row items-start w-full mb-6">
+                <%= image_tag "profile_jordan.png", class: "rounded-full w-12 h-12 mr-5" %>
+                <div class="flex flex-col items-start text-blue-marine">
+                    <p class="font-bold">Jordan G. Trevino</p>
+                    <p>Company Lead</p>
+                </div>
+            </div>
```

Comment:
What  do you think about making the credits section bigger in desktop?

Reply:
I think its worth exploring, I think it would entail a re-design of the whole page for desktop (so that all the sections match in style). 
cc. @LuigiR0jas

Reply:
How does it currently look on desktop?